ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/WronglyTypedContractIdIT.scala

```diff
@@ -41,4 +41,26 @@ final class WronglyTypedContractIdIT extends LedgerTestSuite {
         assertGrpcError(fetchFailure, Code.INVALID_ARGUMENT, "wrongly typed contract id")
       }
   })
+
+  test(
+    "WTMultipleExerciseFails",
+    "Exercising on a wrong type fails after correct exercise in same transaction",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(ledger, party)) =>
```

Comment:
1. I definitively would like a Unit test in daml-lf module. I think there is everything you need in `EngineTest` (this is mess I know :disappointed: )
2. I am inclined to not test at all this engine specific behavior  in the conformance test.

Reply:
I’ve looked at `EngineTest` but it doesn’t have any tests for wrongly typed contract ids atm afaict and it also has no tests that run across transactions in general :disappointed: I’ll see if I can figure something out.

Given that we already have tests for wrongly typed contract ids, I think it’s reasonable to keep this one.

Reply:
220e52fb3b287fd25f626355bc15755008ca9467